### ISSUE TITLE
V5b3 app crashes immediately after start

### ISSUE THREAD

**Issue by ha24-1** [dirkwhoffmann/virtualc64#796]
Hi,
the new version 5.0beta3 crashes immediately after start - please find full stacktrace attached. 
Latest V4.7 (240127) is installed  but not running.
Hope that helps!

Best regards
Heiko

[stackVc64_50b3.txt](https://github.com/user-attachments/files/15838818/stackVc64_50b3.txt)

**Comment by dirkwhoffmann** [comment]
Thanks for reporting! Unfortunately, I cannot see anything in the stack trace, as the executable contains no debug symbols. I'll compile a new version with debug symbols included...

**Comment by dirkwhoffmann** [comment]
Can you give v5.0b4 a try? Hopefully, it still crashes and the stack trace tells us where.

**Comment by ha24-1** [comment]
... of course, would you please provide a dl-link for the b4-installer?

**Comment by ha24-1** [comment]
[Vc64_50b4_stacktrace.txt](https://github.com/user-attachments/files/15839454/Vc64_50b4_stacktrace.txt)

Et voilà !   :-)

**Comment by dirkwhoffmann** [comment]
It crashes inside the GamePadManager. The culprit is line `return gamePad.icon!`

```Swift
   func icon(slot: Int) -> NSImage {

        if let gamePad = gamePads[slot] {

            if gamePad.isMouse && parent.config.mouseModel >= 3 {
                return NSImage(named: "devPaddleTemplate")!
            } else {
                return gamePad.icon!
            }
        }

        return NSImage(named: "devGamepad1Template")!
    }
```

**Comment by dirkwhoffmann** [comment]
The USB device crash should be fixed in v5.0b5. However, your 4.7 stack report indicates another issue on your machine. v5.0b5 does still contain debug symbols. Thus, if it continues crashing, the stack report might tell us where.

**Comment by ha24-1** [comment]
... thank you so much, it works! 
(V4.7 issues doesn't matter - now we have V5 ;-) )
